cyberdrop_dl/managers/config_manager.py

```diff
@@ -143,7 +143,9 @@ def _verify_settings_config(self) -> None:
         self.settings_data["Logs"]["webhook_url"] = str(self.settings_data["Logs"]["webhook_url"])
         self.settings_data["Sorting"]["sort_folder"] = Path(self.settings_data["Sorting"]["sort_folder"])
         self.settings_data["Sorting"]["scan_folder"] = (
-            Path(self.settings_data["Sorting"]["scan_folder"]) if self.settings_data["Sorting"]["scan_folder"] else None
+            Path(self.settings_data["Sorting"]["scan_folder"])
+            if self.settings_data["Sorting"]["scan_folder"] != "None"
+            else None
```

Comment:
This will still throw an error when `scan_folder` is actually `None` and not `"None"`. It will try to convert `None`  to a Path, `Path(None)`. 

`None` is a valid value, is actually the default. Needs a check for both (`None` and `"None"`). Changing the default to an empty string will also solve it cause it will always be a string but it may cause problems in other parts of the code.

Reply:
Having it set to none didn't throw an error in my testing. `None`, even without quotes, is interpreted by the yaml reader as `"None"`, while `null` is interpreted as `None`.

Reply:
Yeah now it's giving me an error 😿 The update that I pushed doesn't work. Will fix when I finish dinner.